### PR TITLE
Openhelpd 54 create message board

### PR DESCRIPTION
**Pull request by miltontony**
screen shot 2016-08-29 at 10 20 43

### CODE REVIEW

**Comment by miltontony** [comment]
@codiebeulaine ready for review (also @jpaidoussi @rudigiesler )

File: casepro/msg_board/tests.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+from django_comments.models import Comment
+from django_comments.forms import CommentForm
+from casepro.msg_board.models import PinnedComment
+
+
+class CommentTest(BaseCasesTest):
+    def setUp(self):
+        super(CommentTest, self).setUp()
+        self.client = Client()
+        self.ann = self.create_contact(self.unicef, 'C-001', "Ann",
```
Comment (codiebeulaine) [inline casepro/msg_board/tests.py]: @miltontony do we need self.ann here?

File: casepro/msg_board/views.py [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+
+
+class MessageBoardView(OrgPermsMixin, SmartTemplateView):
+    template_name = 'msg_board/comment_list.haml'
+    permission = 'orgs.org_home'
+
+
+class CommentsView(OrgPermsMixin, SmartListView):
+    """
+    JSON endpoint for fetching case actions and messages
```
Comment (codiebeulaine) [inline casepro/msg_board/views.py]: @miltontony endpoint for comments?

**Comment by jpaidoussi** [comment]
+1

**Pull request merged**